packages/terra-popup/src/Popup.jsx

```diff
@@ -0,0 +1,370 @@
+import React from 'react';
```

Comment:
I am wondering if it would be good to move the positioning/offset logic into its own `popupUtils` file.